pkg/generators/builder/testdata/d/d.go

```diff
@@ -8,3 +8,6 @@ import (
 type DPolicyRule struct {
 	e.MockPolicyRule
 }
+
+// +kanopy:builder=true
+type AliasType e.AliasToString
```

Comment:
Should a test be added for the case where an aliased primitive type is not enabled, to check that it is not generated?

Reply:
This is added.